code/modules/item_worth/materials.dm

```diff
@@ -1,6 +1,11 @@
 /material
 	var/value = 1
 
+/material/Destroy(force)
+	stack_trace("Some genius is trying to delete a /material ...")
```

Comment:
```suggestion
	stack_trace("Someone tried to destroy a /material.")
```

Reply:
The proc is called Destroy, but it's triggered by QDEL, also Destroy sounds more like someone ingame shot it until it got destroyed, which is not the case.

See eg. `code\datums\weakref.dm`

Reply:
```suggestion
	stack_trace("Someone tried to delete a /material.")
```
works too.